src/components/thumbnail/tab-panel-documents-section.tsx

```diff
@@ -83,21 +83,24 @@ export const TabPanelDocumentsSection = observer(({ tab, section, index, numOfSe
                                   onSelectNewDocument, onSelectDocument, onDocumentDragStart,
                                   onDocumentStarClick, onDocumentDeleteClick }: IProps) => {
     const { user } = stores;
-    const showNewDocumentThumbnail = section.addDocument && !!onSelectNewDocument;
+    const showNewDocumentThumbnail = section.addDocument && !!onSelectNewDocument
+                                      && section.type !== ENavTabSectionType.kProblemDocuments;
```

Comment:
Out of curiosity, is the latter test necessary? Seems like `section.addDocument` should never be true for problem documents, in which you'd never get to the `section.type` test. Unless you're just protecting against improperly authored content and maybe that's reason enough. 😉

Reply:
At some iteration, it was showing because I was trying to get New Workspace to come up in the nav panel side.